src/component/button/PermalinkButton/PermalinkButton.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+
+import SimpleButton from '@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton';
+import Window from '@terrestris/react-geo/dist/Window/Window';
+
+import { TooltipPlacement } from 'antd/lib/tooltip';
+import Button, { ButtonProps } from 'antd/lib/button';
+import { message } from 'antd';
+
+import { PermalinkUtil } from '../../../util/PermalinkUtil/PermalinkUtil';
+
+import './PermalinkButton.css';
+
+
+interface DefaultPermalinkButtonProps {
+  type: 'default' | 'primary' | 'ghost' | 'dashed' | 'danger' | 'link';
```

Comment:
Since antd 4 `danger` is not a type option anymore, it has moved to an own prop `danger`.